cfme/tests/configure/test_remote_server_tabs.py

```diff
@@ -32,7 +32,7 @@ def test_remote_server_advanced_config(temp_appliance_preconfig, request,
         casecomponent: Configuration
     """
     appliance = temp_appliance_preconfig
-    remote_server = appliance.server.slave_servers[0]
+    remote_server = appliance.server.other_servers[0]
```

Comment:
This is `secondary_servers` now yes?

Reply:
This wasn't caught by PRT because of limited test collection, please include more tests in your PRT string.